### DISCUSSION TITLE
[scope] Partial ratings not displaying correctly inside a card content

### DISCUSSION THREAD

**Discussion by tannguyendeveloper** [fomantic/Fomantic-UI#1566] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Bug Report

Partial ratings are not displaying inside of .card .content.  The number of stars shown is rounded up.

## Steps to reproduce

1. Add a star rating inside of a .card .content with the data-rating = 3.5

## Expected result

Expected 3.5 stars when the data-rating is set to 3.5

## Actual result

Got 4 Stars displayed

## Testcase

https://jsfiddle.net/3kzhjL19/1/

## Version

Fomatic 2.8.6
jQuery 3.3.1

**Comment by lubber-de** [comment]
Confirmed, it's related to the old default behavior of ratings before FUI 2.7.0
If you for example use another icon, then it's working. See https://jsfiddle.net/lubber/tbu7op19/2/
We'll fix this

**Comment by TheJltres** [comment]
Hi!

I have seen that card content overrides some values that rating is using.
![image](https://user-images.githubusercontent.com/23702867/87232672-1d8b6300-c3c1-11ea-8864-75d4a41172db.png)

If you disable all them it works fine.
![Peek 11-07-2020 21-55](https://user-images.githubusercontent.com/23702867/87232691-457ac680-c3c1-11ea-8afb-01aae41c80fb.gif)

I think this override is done by: https://fomantic-ui.com/views/card.html#approval
There should be an error with "heart" too, but in CSS name is "like" so its different and there is no override.

**Comment by lubber-de** [comment, marked as answer]
It seems, the solution is much easier. Just add a color  `ui yellow rating` as needed since FUI 2.7 then it's working even with the default star .
https://jsfiddle.net/lubber/tbu7op19/5/

**Comment by lubber-de** [comment]
Closing, because adding a color is needed anyway and solves the issue out of the higher css specificity
